e2e-polybft/e2e/acls_test.go

```diff
@@ -0,0 +1,365 @@
+package e2e
+
+import (
+	"math/big"
+	"testing"
+
+	"github.com/0xPolygon/polygon-edge/contracts"
+	"github.com/0xPolygon/polygon-edge/e2e-polybft/framework"
+	"github.com/0xPolygon/polygon-edge/helper/hex"
+	"github.com/0xPolygon/polygon-edge/state/runtime/addresslist"
+	"github.com/0xPolygon/polygon-edge/types"
+	"github.com/stretchr/testify/require"
+	"github.com/umbracle/ethgo/wallet"
+)
+
+// Contract used as bytecode
+
+/*
+pragma solidity ^0.5.5;
+pragma experimental ABIEncoderV2;
+
+contract Sample {
+	address public constant allowList = 0x0200000000000000000000000000000000000000;
+
+	function readAllowList(address addr) public returns (uint256) {
+		(bool success, bytes memory returnData) = allowList.call(abi.encodeWithSignature("readAllowList(address)", addr));
+		(uint256 val) = abi.decode(returnData, (uint256));
+		return val;
+	}
+}
+*/
+
+func TestE2E_AllowList_ContractDeployment(t *testing.T) {
```

Comment:
Are we able to unify these tests and run them against the same cluster? Would it be hard to accomplish, because of state needs to be resetted at the end of the each test etc.?

Reply:
Exactly